shared/src/main/java/com/couchbase/lite/PropertiesImpl.java

```diff
@@ -65,10 +73,22 @@ public int getInt(String key) {
     }
 
     @Override
-    public double getDouble(String key) {
+    public float getFloat(String key) {
         if (properties != null)
-            return (double) properties.get(key);
+            return (float) properties.get(key);
```

Comment:
If the value is not number such as String or Map, will it throw the ClassCastException when casting to float?